services/movie.py

```diff
@@ -0,0 +1,37 @@
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
```

Comment:
You should indicate that this argument is optional in type annotation, and make changes in arguments that have default value in your code.